libs/geography/src/drivers/magento/transforms/responses/country.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+
+import { DaffMagentoSubdivisionTransformer } from './subdivision.service';
+import { MagentoCountry } from '../../models/responses/country';
+import { DaffCountry } from '../../../../models/country';
+
+/**
+ * Transforms magento carts into an object usable by daffodil.
+ */
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffMagentoCountryTransformer {
+  constructor(public subdivisionTransformer: DaffMagentoSubdivisionTransformer) {}
+
+  private transformSubdivisions(country: MagentoCountry) {
+    return {
+      subdivisions: country.available_regions
+        ? country.available_regions.map(region => this.subdivisionTransformer.transform(region))
+        : []
+    }
+  }
+
+  /**
+   * Transforms the MagentoCountry from the magento country query into a DaffCountry.
+   */
+  transform(country: MagentoCountry): DaffCountry {
+    return country ? {
+      ...{magento_country: country},
+
+      ...this.transformSubdivisions(country),
```

Comment:
I don't think we need a spread here, you could just set it to the prop directly and simplify the transformSubdivisions

Reply:
changed